includes/Story_Post_Type.php

```diff
@@ -321,13 +299,12 @@ public static function admin_enqueue_scripts( $hook ) {
 			[
 				'id'     => 'edit-story',
 				'config' => [
-					'allowedVideoMimeTypes'          => $allowed_video_mime_types,
-					'allowedPageAttachmentPostTypes' => $post_types,
-					'postType'                       => self::POST_TYPE_SLUG,
-					'postThumbnails'                 => $post_thumbnails,
-					'storyId'                        => $story_id,
-					'previewLink'                    => get_preview_post_link( $story_id ),
-					'api'                            => [
+					'allowedVideoMimeTypes' => $allowed_video_mime_types,
```

Comment:
Just noticed that `allowedVideoMimeTypes` isn't implemented. This should be changed [here](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/library/mediaLibrary.js#L134-L136).

We should also maybe add `allowedImageMimeTypes` and maybe  `allowedAudioMimeTypes`

Reply:
Audio upload is not yet a thing, but I can change it for video and images.

Reply:
Done now in 09112acf5058dc6410e5f52bea8471beb4aa8915